content/pt-br/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,472 @@
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+Se você implantar um driver [Interface de armazenamento de contêineres (CSI)](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+que suporta o `VOLUME_MOUNT_GROUP` `NodeServiceCapability`, o
+processo de definir a propriedade e permissões de arquivos com base no
+`fsGroup` especificado no `securityContext`, será realizado pelo driver CSI em vez do Kubernetes, desde que o `DelegateFSGroupToCSIDriver` Kubernetes
+`feature gate` esteja habilitada. Nesse caso, contanto que o Kubernetes não realize nenhuma
+mudança de propriedade e permissão, `fsGroupChangePolicy` não entra em vigor e
+conforme especificado por CSI, é esperado que o driver monte o volume com `fsGroup` fornecido, resultando em um volume que pode ser lido/gravado pelo
+`fsGroup`.
+
+Por favor, consulte o [KEP](https://github.com/gnufied/enhancements/blob/master/keps/sig-storage/2317-fsgroup-on-mount/README.md)
+e a descrição do campo `VolumeCapability.MountVolume.volume_mount_group`
+na [especificação CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md#createvolume)
+para maiores informações.
+
+## Defina o contexto de segurança para um Contêiner
+
+Para especificar configurações de segurança a um Contêiner, inclua o campo `securityContext` no manifesto do Contêiner. O campo `securityContext` é um objeto
+[SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core).
+Configurações de segurança que você especifica para um contêiner se aplicam apenas ao contêiner individual, e substituem as configurações feitas no nível do Pod quando
+há sobreposição. As configurações de Contêiner não afetam os Volumes do Pod.
+
+Aqui está o arquivo de configuração para um pod que possui um contêiner. Ambos, o Pod
+e o Contêiner, tem um campo `securityContext`:
+
+{{< codenew file="pods/security/security-context-2.yaml" >}}
+
+Crie o Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/security/security-context-2.yaml
+```
+
+Verifique se o Contêiner do Pod está executado:
+
+```shell
+kubectl get pod security-context-demo-2
+```
+
+Abra um shell no Contêiner em execução:
+
+```shell
+kubectl exec -it security-context-demo-2 -- sh
+```
+
+No seu shell, liste os processos em execução:
+
+```shell
+ps aux
+```
+
+A saída mostra que os processos estão em execução como usuário 2000. Este é o valor
+do `runAsUser` especificado para o Contêiner. Ele sobrepõe o valor 1000 que é especificado para o Pod.
+
+```
+USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
+2000         1  0.0  0.0   4336   764 ?        Ss   20:36   0:00 /bin/sh -c node server.js
+2000         8  0.1  0.5 772124 22604 ?        Sl   20:36   0:00 node server.js
+...
+```
+
+Saia do seu shell:
+
+```shell
+exit
+```
+
+## Defina recursos para um Contêiner
+
+Com [recursos Linux](https://man7.org/linux/man-pages/man7/capabilities.7.html),
```

Comment:
I'd keep capabilities here. You could stylize it as italic (i.e. _capabilities_) to give the reader the sense that this comes from a foreign language.